cnew.py

```diff
@@ -87,7 +89,7 @@ def chekroom(strg):
     else:
         try:
             geee = int(strg, 16)
-            return True
+            return strg.isupper()
```

Comment:
A truly intelligent way of combining boolean return for both upper-case check and hexadecimal compliance.

Reply:
Thanks man 😊

Reply:
Looks much better now.